components/sections/CompanyOverview/index.jsx

```diff
@@ -0,0 +1,109 @@
+import { useEffect, useState } from "react";
+import ArrowIcon from "../../ui/ArrowIcon";
+import Overview from "./Overview";
+import { motion as Motion } from "framer-motion";
+import companyTabs from "./../../../data/companyTabs";
+import Wrapper from "../../Wrapper";
+
+const CompanyOverview = () => {
+  const [activeTab, setActiveTab] = useState(companyTabs[0]);
+  const [activeTabIndex, setActiveTabIndex] = useState(0);
+  const changeActiveTabMobile = (tab) =>
+    tab === activeTabIndex ? setActiveTabIndex(null) : setActiveTabIndex(tab);
+
+  useEffect(() => {
+    const handleResize = () => {
+      if (window.innerWidth >= 640 && !activeTabIndex) {
```

Comment:
You should avoid abstract values. `640` should be described by a name.

```suggestion
      if (window.innerWidth >= MAX_MOBILE_WIDTH && !activeTabIndex) {
```